sparkplug/config/consumer.py

```diff
@@ -87,6 +88,8 @@ def __init__(self, connection, consumer):
         self._heartbeater = Heartbeater(connection)
 
     def __call__(self, msg):
+        if SignalHandler.should_terminate:
+            _log.warning("Terminating Gracefully")
```

Comment:
Seems like this should do a bit more no? In this current configuration doesn't this just warn log and then continue as normal?